App/Species/KineticSpecies.lua

```diff
@@ -466,6 +467,25 @@ function KineticSpecies:createSolver(field, externalField)
          basis  = self.basis,
       }
    end
+   if self.nonconPositivity then
+      self.prePosM0     = self:allocMoment()
+      self.postPosM0    = self:allocMoment()
+      self.delPosM0     = self:allocMoment()
+      self.intDelPosM0  = DataStruct.DynVector{numComponents = 1}
+      self.intPosM0     = DataStruct.DynVector{numComponents = 1}
+      self.calcIntPosM0 = Updater.CartFieldIntegratedQuantCalc {
+	    onGrid        = self.confGrid,
+	    basis         = self.confBasis,
+	    numComponents = 1,
+	    quantity      = "V",
+	    timeIntegrate = timeIntegrate,
```

Comment:
Should this line be removed? I don't see where the `timeIntegrate` on the right is defined.

Reply:
Also meant to delete that. Thanks for catching.